adapters/http-vertx-base/src/main/java/org/eclipse/hono/adapter/http/HttpAdapterMetrics.java

```diff
@@ -37,4 +38,15 @@ void incrementUndeliverableHttpMessages(final String resourceId, final String te
         counterService.increment(getPrefix() + MESSAGES + mergeAsMetric(resourceId, tenantId) + UNDELIVERABLE);
     }
 
+    void incrementCommandDeliveredToDevice(final String tenantId) {
+        counterService.increment(METER_PREFIX + getPrefix() + mergeAsMetric(COMMANDS, tenantId, "device", "delivered"));
+    }
+
+    void incrementNoCommandReceivedAndTTDExpired(final String tenantId) {
+        counterService.increment(METER_PREFIX + getPrefix() + mergeAsMetric(COMMANDS, tenantId, "ttd", "expired"));
```

Comment:
Same as above.

Reply:
I have pulled the COMMANDS to one level up to the Metrics.